config/prod/xschildw-stack-tags-alias.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision a Windows EC2 instance
+template_path: remote/managed-ec2-win.yaml
+stack_name: xschildw-test-stack-tags
+
+stack-tags: &tags
+  Department: "IT"
+  Project: "Infrastructure"
+  OwnerEmail: "x.schildwachter@sagebase.org"
+
+parameters:
+  <<: *tags
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
```

Comment:
due to the provisioner's evolution you'll want to set these parameters to..
```
  VpcName: "sandcastlevpc"
  KeyName: "sandbox"
  AMIId: "ami-04ca2d0801450d495"    # Windows_Server-2019-English-Full-Base-2019.07.12
```

Reply:
Thx!